packages/beacon-node/test/unit/chain/stateCache/persistentCheckpointsCache.test.ts

```diff
@@ -10,7 +10,7 @@ import {ShufflingCache} from "../../../../src/chain/shufflingCache.js";
 import {testLogger} from "../../../utils/logger.js";
 import {getTestDatastore} from "../../../utils/chain/stateCache/datastore.js";
 import {CheckpointHex} from "../../../../src/chain/stateCache/types.js";
-import {toCheckpointHex} from "../../../../src/chain/index.js";
+import {FIFOBlockStateCache, toCheckpointHex} from "../../../../src/chain/index.js";
 
 describe("PersistentCheckpointStateCache", function () {
```

Comment:
Do you need to add a test case for falling back to use seed state from FIFOBlockStateCache if not available in PersistentCheckpointStateCache cache?

Reply:
not possible for now, need to wait for proposer boost reorg in order to overwrite `getProposerHead()` in `ReorgedForkchoice` https://github.com/ChainSafe/lodestar/pull/6674/files#diff-b005dbf9536b2e2e5b38eadbd41b4fdc6fd3b22b1daed91c902895ddcfd3e84bR252